InterfaceFirmware/include/hw/UART.hpp

```diff
@@ -7,13 +7,14 @@
 #include <vector>
 #include <cctype>
 #include <cstdlib>
+#include <string>
 
 extern class UART* irq_userptrs[2];
 
 void on_uart0_irq();
 void on_uart1_irq();
 
-constexpr std::size_t RX_BUFFER_LEN = 32;
+constexpr uint8_t RX_BUFFER_LEN = 32;
```

Comment:
Eventhough uint8_t here is fiiine, I think it is just asking for trouble if it has to be increased at some point.

Reply:
feel free to ignore btw